modyn/storage/src/internal/database/sql/SQLUpdateDataset.sql

```diff
@@ -1,4 +1,4 @@
-R"(UPDATE datasets 
+R"(UPDATE datasets
```

Comment:
Ah, I don't want to vouch for this working. Maybe we want to exclude .sql files from removing whitespaces. if newlines are not equal to a white space here, this would mean `UPDATE datasetsSET base_path` which would be problematic.

Reply:
Right, we are using `.sql` files in the most unintuitive way through C++ preprocessor imports😅 In April I spent a couple of hours searching for a better way, but didn't find anything that C++ would accept. Seemingly, you cannot load files in compile time / constexpr functions😅

Reply:
Excluding directories is possible :)

Reply:
Doesn't seem to be harmful though